src/shared/mount-setup.c

```diff
@@ -148,6 +148,22 @@ bool mount_point_ignore(const char *path) {
         return false;
 }
 
+bool memory_recursiveprot_supported(void) {
+        int r;
+
+        r = mount_option_supported("cgroup2", "memory_recursiveprot", "yes");
```

Comment:
3rd arg should be NULL, as this mount option does not take any parameter

Reply:
fixed.